### PR TITLE
Applingua Localizations

### PR DESCRIPTION
**Pull request by applingua**
German localization for Homey

### CODE REVIEW

File: .homeycompose/locales/de.en.rf.pair.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
+{
```
Comment (WeeJeWel) [inline .homeycompose/locales/de.en.rf.pair.json]: Please rename file to `de.rf.pair.json` (without `en`)

File: .homeycompose/locales/de.en.rf.pair.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  "imitate": {
```
Comment (WeeJeWel) [inline .homeycompose/locales/de.en.rf.pair.json]: Is there a reason this file isn't translated?

Comment (WeeJeWel) [reply .homeycompose/locales/de.en.rf.pair.json]: Ah, I see the file below. Then delete this file.

**Comment by applingua** [comment]
@WeeJeWel Our master repo on our fork doesn't have these de.en files, only de.rf, etc. I can't see where they're coming from :( Should you reject the merge and I try again?

File: .homeycompose/locales/de.en.rf.pair.json (hunk 1 above)
Comment (applingua) [reply .homeycompose/locales/de.en.rf.pair.json]: This file can just be deleted. It's from a previous sync with our software, the correctly named and translated file is below.

**Comment by applingua** [comment]
Issue resolved if you wish to proceed.

**Pull request merged**